app/src/main/java/com/github/kulebin/myfoursquareapp/VenueDetailFragment.java

```diff
@@ -21,6 +21,7 @@
 
     public static final String TAG = VenueDetailFragment.class.getSimpleName();
     private static final String VENUE_ID = "venueId";
+    private static final String IS_TABLET_MODE = "isTabletMode";
```

Comment:
check names, probably we can have small tablets to have a detail when lanscape and no detail when portrait